street_network/src/lanes/mod.rs

```diff
@@ -32,6 +32,11 @@ pub enum LaneType {
     Construction,
     LightRail,
     Buffer(BufferType),
+    /// Some kind of pedestrian-only path unassociated with a road
```

Comment:
Very related to #77. I'm not happy with this classification as "only pedestrians" and "both pedestrians and cyclists." Things like living streets or alleyways, where motor vehicles are also allowed, aren't captured yet. Maybe we just need one shared use lane type, with a way to specify the legal or cultural priority between vehicles. For instance, I know many streets in the US without pavement, where it's effectively shared use and motor vehicles get priority.

Reply:
I think when it comes down to it, every lane is a list of specific designations and restrictions. There are also classifications, informed by the locale, I suppose, like "only pedestrians", "bike lane" or "bus lane (bikes allowed)" that inform markup decisions.

Reply:
I like this way of thinking about it. Going to start a fresh issue on this soon to consolidate all of the thoughts around this...